### PR TITLE
added a tutorial for the Maps module

### PR DESCRIPTION
**Pull request by ShuaibNuruddin**
This pull request addresses issue #390 by introducing a tutorial for the maps module. I welcome feedback to enhance the clarity and effectiveness of the tutorial content.

### CODE REVIEW

File: docs/tutorial.rst [hunk 1]
```diff
@@ -388,4 +388,48 @@ Let's do the bootstrap test on the two categories.
 
 Drawing Maps
 ------------
-To come.
+The main class in the maps module is the Map class. In this code we create a default map. Maps can be displayed or converted to html.
```
Comment (adnanhemani) [inline docs/tutorial.rst:391]: "In this code we create a default map" -> "In this code we create a default Map" (to distinguish against Python maps)

**Review by adnanhemani — CHANGES_REQUESTED** [review]
Very nicely done! Just the one minor comment and then we should be ready to merge and deploy.

**Comment by ShuaibNuruddin** [comment]
Thank you for your feedback. I have changed it from map to Map.

**Review by adnanhemani — APPROVED** [review]

**Comment by adnanhemani** [comment]
Just merged! Once again, amazing job on this - looking forward to future contributions :)

**Pull request merged**